changelog.yaml

```diff
@@ -51,6 +51,17 @@
 - version: NEXT
   date: TBD
   changes:
+
+  - type: enhancement
+    impact: minor
+    title: Update Jenkinsfile-runner image version
+    description: |-
+      Update Jenkinsfile-runner image to a newer version.
```

Comment:
Please add more details what changed in JFR image with some link to the changelog.

Reply:
I added a link for the changes in the new JFR image